File: Sorting/Quick Sort/QuickSort.py
def quick_sort(arr):
    if len(arr) <= 1:
        return arr
    pivot = arr[len(arr) - 1]
    left = [x for x in arr[:-1] if x < pivot]
    right = [x for x in arr[:-1] if x >= pivot]
    return quick_sort(left) + [pivot] + quick_sort(right)

if __name__ == "__main__":
    n = int(input("Enter the number of elements: "))
    arr = list(map(int, input(f"Enter {n} elements: ").split()))
    sorted_arr = quick_sort(arr)
    print("Sorted array:", *sorted_arr)

File: Sorting/Quick Sort/QuickSort.js
function quickSort(arr) {
    if (arr.length <= 1) {
        return arr;
    }
    let pivot = arr[arr.length - 1];
    let left = arr.slice(0, -1).filter(x => x < pivot);
    let right = arr.slice(0, -1).filter(x => x >= pivot);
    return [...quickSort(left), pivot, ...quickSort(right)];
}

const readline = require('readline');
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.question("Enter numbers separated by spaces: ", input => {
    let arr = input.split(" ").map(Number);
    console.log("Sorted array:", quickSort(arr));
    rl.close();
});
